packages/test/test-end-to-end-tests/src/test/summaries.spec.ts

```diff
@@ -307,6 +323,82 @@ describeNoCompat("Summaries", (getTestObjectProvider) => {
 		);
 	});
 
+	it("full initialization of data object should not happen by default", async () => {
+		const TestDataObjectType1 = "@fluid-example/test-dataStore1";
+		const dataStoreFactory1 = new DataObjectFactory(
+			TestDataObjectType1,
+			TestDataObject1,
+			[],
+			[],
+		);
+		const registryStoreEntries = new Map<string, Promise<IFluidDataStoreFactory>>([
+			[dataStoreFactory1.type, Promise.resolve(dataStoreFactory1)],
+		]);
+		const containerRuntimeFactoryWithDefaultDataStore =
```

Comment:
I am curious why you get a specific version of the container runtime factory here.
This will result in only the current version of container runtime factory created even when the test runs in back-compat mode.

Reply:
Yea. You are right. I don't need it to be current version. Default version works fine.